README.md

```diff
@@ -1 +1,44 @@
-The documentation is on [spilo.readthedocs.org](http://spilo.readthedocs.org/).
+## Spilo: HA PostgreSQL Clusters with Docker
+
+Spilo is a Docker image that provides a resilient, High Available PostgreSQL cluster (HA-cluster) that you can configure and start within minutes. Spilo can run several clusters at one time, with one cluster acting as a master and the others as slaves. Its name derives from სპილო [spiːlɒ], the Georgian word for "elephant." 
```

Comment:
Spilo provides PostgreSQL and Patroni bundled together. Multiple Spilos can create a HA PostgreSQL cluster. For this, all participating Spilos should be started with the same DCS (distributed configuration system) and cluster name parameters.